src/_reducers/HelloWorldReducer.js

```diff
@@ -0,0 +1,15 @@
+import StateStorage from '../_store/StateStorage';
+
+const SayHello = 'Hello';
+const SayHi = 'Hi';
+const NoSayHello = '';
+
+const initialState = new Map({subject: 'World'});
+
+export default (state=initialState, action=null) => {
```

Comment:
I don't think action = null is necessary, the default will be undefinied which is almost equivalent.
The rest looks correct.